src/main/java/com/band/api/AppConfiguration.java

```diff
@@ -24,22 +24,23 @@ public PasswordEncoder PasswordEncoder() {
 
     @Bean
     public GraphQLErrorHandler errorHandler() {
-        return new GraphQLErrorHandler() {
-            @Override
-            public List<GraphQLError> processErrors(List<GraphQLError> errors) {
-
-                return Stream.concat(
-                        errors.stream()
-                                .filter(this::isClientError),
-                        errors.stream()
-                                .filter(Predicate.not(this::isClientError))
-                                .map(GraphQLErrorAdapter::new)
-                ).collect(Collectors.toList());
-            }
-
-            private boolean isClientError(GraphQLError error) {
-                return !(error instanceof ExceptionWhileDataFetching || error instanceof Throwable);
-            }
-        };
+        return (errors)->processErrors(errors);
```

Comment:
Add spaces around the arrow.

Reply:
I disagree, it acts more like a dot operator than an arithmetic operator.